CODEOWNERS

```
@@ -125,5 +125,11 @@ go.mod @fleetdm/go
 ##############################################################################################
 /.github/workflows/ @lukeheath @georgekarrv @sharon-fdm @mostlikelee
 
+##############################################################################################
+# 🌐 Fleet-maintained apps
+##############################################################################################
+/ee/maintained-apps/ @eashaw
```

```suggestion
/ee/maintained-apps/outputs/apps.json @eashaw
```

Drilling down to this level ensures gating of changes that might cause the app library to render titles without icons (though IMO we should check icons into the repo...we should have a sync meeting about this), while allowing individual manifests to be updated with the latest versions/scripts/install/patching queries, which we're now all managing via the repo (and need to stay on top of so customers using FMA don't wind up on outdated app versions).

This also allows us to revise the code that builds the manifests, which won't affect the website but _does_ affect our ability to auto-generate the correct install scripts etc.